tools/ci/after_install.sh

```diff
@@ -0,0 +1,9 @@
+if [ "$TEST_SUITE" = "spec" ]; then
+  bundle exec codeclimate-test-reporter;
+fi
+
+# Collapse Travis output https://github.com/travis-ci/travis-ci/issues/2158
+echo "travis_fold:start:GEMFILE_LOCK"
+echo "Gemfile.lock: "
+cat Gemfile.lock
```

Comment:
Nice idea, but you should rather use [bundle show](http://bundler.io/v1.1/bundle_show.html) as the lockfile contains a lot of duplicities.